### DISCUSSION TITLE
Get rid of 2023 UI refresh

### DISCUSSION THREAD

**Discussion by CypherPotato** [Eloston/ungoogled-chromium#2870] [General]
![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/17441745/bc26d810-2459-491f-b754-8ec768809f96)

Just wondering if someone doens't want the new Chrome UI, just launch the `chrome.exe` with the flag `--disable-features=CustomizeChromeSidePanel` and you're ready to go.

**Comment by Rexadev** [comment]
How to lunch with flags with Portable chrlauncher ? I use Marmaduke Windows build
